webapp/packages/plugin-data-viewer/src/TableViewer/TableHeader/TableWhereFilter.tsx

```diff
@@ -36,29 +37,46 @@ export const TableWhereFilter: PlaceholderComponent<ITableHeaderPlaceholderProps
   resultIndex,
 }) {
   const translate = useTranslate();
```

Comment:
too messy, logic should be the next:
if we get `requestFilter` and there no `filterConstraints` we show it as an actual filter value
if we change value we execute `constraints.deleteFiltersFromConstraints();` (without any check for existence)
when we change value, we set it in `model.source.options.whereFilter` directly